drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb

```diff
@@ -0,0 +1,327 @@
+        max_date = [enrollment.exit&.ExitDate, report.filter.end, Date.current].compact.min
+        (max_date - enrollment.EntryDate).to_i
+      end
+      report_item.lot = lot
+      end_date = [enrollment.exit&.ExitDate, report.filter.end, Date.current].compact.min
+      max_service = if project_tracking_method == 3 # NbN ES
+        # most recent service, or start date if no service
+        en_services.max_by(&:DateProvided)&.DateProvided || enrollment.EntryDate
+      elsif enrollment.project.project_type_to_use == 4 # SO
+        # max CLS for SO, or start date if no CLS
+        en_cls.max_by(&:InformationDate)&.InformationDate || enrollment.EntryDate
+      else
+        # min of exit date, report end, current date
+        end_date
+      end
+      # count the days between the end of the earlier of the reporting end date or exit date and the most-recent service or the entry date
+      report_item.days_since_last_service = (end_date - max_service).to_i
+      report_item
+    end
+
+    def self.sections
+      {
+        disabling_condition_issues: {
+          title: 'Disabling Condition',
+          description: 'Disabling condition is an invalid value',
+          limiter: ->(item) {
+            ! HUD.no_yes_reasons_for_missing_data_options.key?(item.disabling_condition)
+          },
+        },
+        hoh_validation_issues: {
+          title: 'Relationship to Head of Household',
+          description: 'Relashionship to head of household is an invalid value',
+          limiter: ->(item) {
+            ! HUD.relationships_to_hoh.key?(item.relationship_to_hoh)
+          },
+        },
+        living_situation_issues: {
+          title: 'Living Situation',
+          description: 'Living situation is an invalid value',
+          limiter: ->(item) {
+            return false if item.living_situation.blank?
+
+            ! item.living_situation.in?(HUD.valid_prior_living_situations)
+          },
+        },
+        exit_date_issues: {
+          title: 'Exit before Entry',
+          description: 'Enrollment exit date must occur after entry date',
+          limiter: ->(item) {
+            item.exit_date.present? && item.exit_date < item.entry_date
+          },
+        },
+        destination_issues: {
+          title: 'Destination',
+          description: 'Destination is an invalid value',
+          limiter: ->(item) {
+            return false if item.exit_date.blank?
+
+            ! HUD.valid_destinations.key?(item.destination)
+          },
+        },
+        unaccompanied_youth_issues: {
+          title: 'Unaccompanied Youth < 12 Years Old',
+          description: 'Youth under 12 are generally expected to be accompanied.  The presence of an unaccompanied youth under 12 may indicate an issue with household data collection',
+          limiter: ->(item) {
+            item.household_max_age.present? && item.household_max_age <= 12
```

Comment:
I think this is HUDs fuzziness in the description, but this implements under the age of 13, right?

Reply:
correct, should be < 12